apps/mobile/src/libs/components/icon-button/icon-button.tsx

```diff
@@ -0,0 +1,20 @@
+import React from "react";
+
+import { MaterialIcon, Pressable } from "~/libs/components/components";
+
+type Properties = {
+	color: string;
+	name: string;
+	onPress: () => void;
+	size: number;
+};
+
+const IconButton: React.FC<Properties> = ({ color, name, onPress, size }) => {
```

Comment:
Let's have a separate Icon component, which will receive `iconName` as a prop. It can also receive `onPress`, in that case we will handle it as a button

Reply:
I think we shouldn't pass onPress to the Icon component directly. We can have two separate components: one - the Icon as a wrapper around the librarys' icons, the second one - the IconButton. We can add color, name and size props to Icon, but IconButton should have a Touchableopacity wrapper with 'hitSlop' prop set.

Reply:
Let's keep IconButton, but we also need a common Icon component, which will receive iconName

Reply:
So it is better to create a separate `Icon` component as a wrapper of `MaterialIcon`, and use it inside a `IconButton` component?

Reply:
Yep, that's right